### ISSUE TITLE
Links do not pass color contrast requirements

### ISSUE THREAD

**Issue by maya** [18F/federalist#1212]
The links do not pass color contrast requirements, it's too light. Seems to fail on both white and gray backgrounds. <img width="1006" alt="screen shot 2017-10-03 at 3 32 30 pm" src="https://user-images.githubusercontent.com/5249443/31152205-f11a825e-a84f-11e7-9b7b-c7f911cbfab3.png">

**Comment by wslack** [comment]
hmmmm. That's interesting; I don't think we changed that recently but it is a different color.

**Comment by wslack** [comment]
@maya opened PR: https://github.com/18F/federalist/pull/1213
